source/components/GlobalNavigation/components/GlobalNavigationNotifications/MarkAllButton.js

```diff
@@ -20,9 +21,9 @@ const MarkAllButton = ({ track, header }) => {
             {
                 header && <p className="wds-notifications__header">{header}</p>
             }
-            <a onClick={onClick} className="wds-notifications__mark-all-as-read">
+            <button onClick={onClick} className="wds-notifications__mark-all-as-read" role="presentation">
```

Comment:
we should remove this eslint rule as I don't think we care about accessibility :/

Reply:
i've already disabled a few, but yes - seems like we need to get rid of a few others as well